.github/workflows/geoserver.yml

```diff
@@ -10,12 +10,15 @@ jobs:
     steps:
     - name: "Checking out"
       uses: actions/checkout@v1
+
     - name: "Checking out submodules"
       uses: textbook/git-checkout-submodule-action@2.0.0
```

Comment:
maybe we could clone only the submodule we need ? 

maybe `git submodule update geoserver/geoserver-submodule` ?

Reply:
yes but still you'll have to deactivate the "all" module, so that you will exclude the modules which are not cloned (in your case here geonetwork):

https://github.com/georchestra/georchestra/blob/master/pom.xml#L893-L916

Hence the `-P-all,module-you-are-interested-in`